src/main/js/helpers/dateline.js

```diff
@@ -144,7 +164,39 @@ const createDateline = (scale, config, canvasSVG) => {
             )
         );
     });
-    translateDateline(scale, config, canvasSVG, getYAxisYPosition);
+    translateDateline(scale, config, canvasSVG, getYAxisYPosition, transition);
+
+    if (
+        config.pan !== undefined &&
+        utils.isBoolean(config.pan.enabled) &&
+        config.pan.enabled &&
+        config.dateline.length > 0
+    ) {
+        const shapeHeightArr = [];
+        const shape = d3.selectAll(`.${styles.datelinePoint}`);
+        shape[0].forEach((element) => {
+            const shapeHeight = element.getBBox().height;
+            shapeHeightArr.push(shapeHeight);
+        });
+        const datelineIndicatorHeight = Math.floor(
+            Math.max(...shapeHeightArr) / 2
+        );
+        canvasSVG
+            .select(`clipPath#${config.datelineClipPathId}`)
+            .selectAll("rect")
+            .attr(
+                "height",
+                getYAxisHeight(config) === 0
+                    ? datelineIndicatorHeight + 20
```

Comment:
Why the arbitrary numbers? 20, 4?

Reply:
because fro gantt the defs for dateline when there is no content the height is zero so since the indicators are little above the axis they are hidden so I have attached the what is least height and y-axis I can provide so that for no content the indicators are visible since dateline function is a common one.

Reply:
I mean if this is Gantt specific, then doing this will affect Graph construct as well. Is that what you want?

Reply:
No it will not affect the graph construct it will work only for gantt because for graph the y-axis will never be zero without any content but for gantt graph it will become zero for without any content.

Reply:
I am just concerned adding magic numbers like this would affect adversely when other parameters like padding are changed. We are already seeing issues with having arbitrary numbers everywhere. 
I am not sure why we are even showing the dateline when there is no content in Gantt. Since unlike Graph Construct, the content in Gantt is what makes the height of the graph. 
* Can you add some screenshots for Gantt with no data and with dateline (Without panning)? 
* Also screenshots with Gantt before and after adding the 20 and 4?

Reply:
![image](https://user-images.githubusercontent.com/53033168/65327617-109f4800-dbd2-11e9-8c34-f29508d8c66b.png)